### PR TITLE
Note out of the box integration with GKE

### PR DESCRIPTION
**Pull request by amujumdar**
Update README to note that tools in this repository are integrated
out of the box with GKE. These tools are not meant for end users.

Change Stackdriver to Cloud Operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-This repo contains tools for integrating [Stackdriver][stackdriverSite] with
-your Kubernetes deployment. Stackdriver provides advanced monitoring and logging
-solution that will allow you to get more insight into your Kubernetes clusters.
+> :exclamation: **Tools in this repository are not meant for 
+> end-users.**
+> It contains source code for tools that are pre-installed in 
+> the GKE clusters.
 
-[stackdriverSite]: https://cloud.google.com/stackdriver/
+Google Cloud Operations suite (fka Stackdriver) provides advanced
```
Comment (xichen2020) [inline README.md:8]: Did you mean to add `[cloudOperationsSite]` here?

**Comment by josephflynch** [comment]
LGTM

**Comment by 44past4** [comment]
LGTM

**Comment by StevenACoffman** [comment]
@amujumdar Just checking, but it appears that https://github.com/GoogleCloudPlatform/k8s-stackdriver/tree/master/custom-metrics-stackdriver-adapter is not applied out of the box on a new autopilot cluster, correct?

**Pull request merged**